tensorflow/lite/python/lite.py

```diff
@@ -1437,11 +1437,11 @@ def convert(self, graph_def, input_tensors, output_tensors):
       output_tensors: List of output tensors.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```

Comment:
```suggestion
      The converted data in serialized format.
```

Please revert this edit. The format of `Returns:` is a description of the returned value, as a noun phrase. It should be read as "this function returns the converted data in serialized format", but with your edit it gets to "this function returns the converted data is in serialized format"